iOS/src/toga_iOS/widgets/slider.py

```diff
@@ -11,37 +19,57 @@ class TogaSlider(UISlider):
 
     @objc_method
     def onSlide_(self, obj) -> None:
-        self.interface.on_change(None)
+        self.impl.on_change()
 
+    @objc_method
+    def onPress_(self, obj) -> None:
+        self.impl.on_press()
+
+    @objc_method
+    def onRelease_(self, obj) -> None:
+        self.impl.on_release()
 
-class Slider(Widget):
+
+class Slider(Widget, toga.widgets.slider.ContinuousSliderImpl):
     def create(self):
         self.native = TogaSlider.alloc().init()
         self.native.interface = self.interface
         self.native.impl = self
 
-        self.native.continuous = True
-        self.native.addTarget(
-            self.native,
-            action=SEL("onSlide:"),
-            forControlEvents=UIControlEventValueChanged,
-        )
+        for method, events in [
+            ("onSlide", UIControlEventValueChanged),
+            ("onPress", UIControlEventTouchDown),
+            (
+                "onRelease",
+                UIControlEventTouchUpInside
+                | UIControlEventTouchUpOutside
+                | UIControlEventTouchCancel,
+            ),
+        ]:
+            self.native.addTarget(
+                self.native, action=SEL(f"{method}:"), forControlEvents=events
+            )
 
         # Add the layout constraints
         self.add_constraints()
 
-    def get_value(self):
+    def get_continuous_value(self):
         return self.native.value
 
-    def set_value(self, value):
-        self.native.setValue_animated_(value, True)
+    def set_continuous_value(self, value):
+        self.native.value = value
```

Comment:
It's entirely cosmetic, but is there a reason to *not* use the animation here?

Reply:
In examples/slider, enabling animation makes the disabled slider lag behind a little bit. On the other hand, it makes the "move to tick on release" behavior of the discrete slider look much better, so I've enabled it.